### DISCUSSION TITLE
Breaking change not being counted as such (no major bump)

### DISCUSSION THREAD

**Discussion by sinedied** [semantic-release/semantic-release#2045] [Commit Convention]
## Current behavior

A commit with the following format:
```
chore: abc

BREAKING CHANGE:
12345
```

Resulted in a patch version bump instead of a major bump.
The breaking change was correctly detected though, as seen in the changelog here: https://github.com/ngx-rocket/scripts/blob/main/CHANGELOG.md

## Expected behavior

A major version bump.

## Environment

- **semantic-release** version: Latest (using `npx semantic-release` on CI)
- CI environment: GitHub actions
- Plugins used: 
```
    "@semantic-release/changelog": "^5.0.1",
    "@semantic-release/commit-analyzer": "^8.0.1",
    "@semantic-release/git": "^9.0.0",
    "@semantic-release/github": "^7.2.0",
    "@semantic-release/npm": "^7.0.9",
    "@semantic-release/release-notes-generator": "^9.0.1"
```
- **semantic-release** configuration: https://github.com/sinedied/semantic-release-npm-github/blob/main/.releaserc.json
- CI logs: https://github.com/ngx-rocket/scripts/runs/3117732752?check_suite_focus=true

**Comment by travi** [comment]
the commit convention [requires](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular#footer) a summary of the breaking change on either the same line as the `BREAKING CHANGE: ` trigger words or separated by a blank line. the commit analyzer strictly follows the convention definition that is defined outside of semantic-release

**Reply by sinedied** [reply]
I wasn't aware of that Angular convention specificity, my bad. What's disturbing though is that the changelog was generated correctly with the breaking change correctly detected.

**Reply by travi** [reply]
the changelog generation is a bit separate and is configured separately. it uses `conventional-changelog` under the hood, which may have slightly different strictness. you can find details here: https://github.com/semantic-release/release-notes-generator
